sdk/communication/Azure.Communication.Chat/tests/SessionRecords/ChatClientsLiveTests/ReadReceiptGSAsyncAsync.json

```diff
@@ -129,75 +141,81 @@
       "ResponseHeaders": {
         "api-supported-versions": "2020-07-20-preview2, 2021-03-07",
         "Content-Type": "application/json; charset=utf-8",
-        "Date": "Fri, 05 Feb 2021 04:50:18 GMT",
-        "MS-CV": "PyIXIjbb80q7x9NvojonUw.0",
+        "Date": "Fri, 12 Feb 2021 03:08:48 GMT",
+        "MS-CV": "S9xC2hGsZEeYzHQocpbTqw.0",
         "Strict-Transport-Security": "max-age=2592000",
         "Transfer-Encoding": "chunked",
-        "X-Azure-Ref": "0i84cYAAAAAA9vMloZGB8TYqn9TnspwnQRVdSMzBFREdFMDYxMAA5ZmM3YjUxOS1hOGNjLTRmODktOTM1ZS1jOTE0OGFlMDllODE=",
+        "X-Azure-Ref": "0QPElYAAAAACMiFrDDnQYTJ4/yHHSQc5TWVZSMzBFREdFMDQwOAA5ZmM3YjUxOS1hOGNjLTRmODktOTM1ZS1jOTE0OGFlMDllODE=",
         "x-ms-client-request-id": "bc512a3c73790c03e8badb9b1da9b890",
-        "X-Processing-Time": "290ms"
+        "X-Processing-Time": "25ms"
       },
       "ResponseBody": {
         "token": "Sanitized",
-        "expiresOn": "2021-02-06T04:50:18.3396411\u002B00:00"
+        "expiresOn": "2021-02-13T03:08:47.9804957\u002B00:00"
       }
     },
     {
-      "RequestUri": "https://chat-sdktester-e2e.communication.azure.com/chat/threads?api-version=2020-11-01-preview3",
+      "RequestUri": "https://sanitized.communication.azure.com/chat/threads?api-version=2020-11-01-preview3",
       "RequestMethod": "POST",
       "RequestHeaders": {
         "Accept": "application/json",
         "Authorization": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c",
```

Comment:
Is this intentional that Authorization header is not sanitized?

Reply:
nope that is a mistake